### PR TITLE
Configuration Restyling

### PR DESCRIPTION
**Pull request by Kanubiz**
Made a lot of initial changes to the configs overall
Currently restyling the shell
Homogenizing certain colors (some colors were exactly the same color/purpose but in a slightly different shade for example MP bars had like 4 different shades of the same blue)
Changed the gradient to top to bottom
Changed some colors to match MaterialUI

IMPORTANT: Need someone to run a code formatting just in case mine is still broken

### CODE REVIEW

File: DelvUI/Config/Tree/ConfigNode.cs [hunk 1]
```diff
@@ -146,19 +148,83 @@ public override void LoadBase64String(string[] importStrings)
...
+            ImGui.PopStyleColor();
+            
+            ImGui.PopStyleColor();
+            ImGui.PopStyleColor();
+            
+            ImGui.PopStyleVar();
+            ImGui.PopStyleVar();
+            ImGui.PopStyleVar();
+            ImGui.PopStyleVar();
+            ImGui.PopStyleVar();
```
Comment (Tischel) [inline DelvUI/Config/Tree/ConfigNode.cs:204]: You can just do **PopStyleColor(14)** and **PopStylevar(5)**

File: DelvUI/Helpers/TextTags.cs [hunk 2]
```diff
@@ -1,100 +1,99 @@
...
+        }
+
+        public static string GenerateFormattedTextFromTags(dynamic actor, string text)
+        {
+            var matches = Regex.Matches(text, @"\[(.*?)\]");
+
+            return matches.Cast<Match>().Aggregate(text, (current, m) => current.Replace(m.Value, ReplaceTagWithString(m.Value, actor)));
+        }
+    }
+}
```
Comment (Tischel) [inline DelvUI/Helpers/TextTags.cs:99]: Is there any real change in this file?

**Review by Tischel — CHANGES_REQUESTED** [review]
See comments below.

Also we should probably stop using the color maps and just straight up use PluginConfigColor and its properties. A small refactor on BarBuilder would be needed. Doesn't need to happen on this PR, but hardcoding the color map keys everywhere is not ideal.

File: DelvUI/Helpers/TextTags.cs (hunk 2 above)
Comment (Kanubiz) [reply DelvUI/Helpers/TextTags.cs:99]: nop

**Pull request merged**